src/components/SocialComment/styles.ts

```diff
@@ -0,0 +1,42 @@
+import styled, { css } from 'styled-components'
+
+import { SocialCommentsWrapper } from 'templates/Home/styles'
+
+export const Wrapper = styled.div`
+  ${({ theme }) => css`
+    display: flex;
+    flex-direction: column;
+    gap: ${theme.spacings.base};
+
+    padding: ${theme.spacings.lg} ${theme.spacings.base};
+
+    background-color: ${theme.colors.backgroundLight};
+    border-radius: ${theme.border.radius};
+
+    ${SocialCommentsWrapper} & {
+      margin-bottom: ${theme.grid.gutter};
+    }
+  `}
+`
+
+export const Comment = styled.p`
```

Comment:
Mais um que caiu na "pegadinha" da semântica. O ideal para esse tipo de quote, é usar exatamente quote.

https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/